plugins/AIActivity/Reports/Get.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license https://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+declare(strict_types=1);
+
+namespace Piwik\Plugins\AIActivity\Reports;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Piwik;
+use Piwik\Plugin\Report;
+use Piwik\Plugins\AIActivity\FeatureFlags\AIActivity;
+use Piwik\Plugins\AIActivity\Metrics;
+use Piwik\Plugins\CoreVisualizations\Visualizations\JqplotGraph\Evolution;
+use Piwik\Plugins\FeatureFlags\FeatureFlagManager;
+use Piwik\Report\ReportWidgetFactory;
+use Piwik\Widget\WidgetsList;
+
+class Get extends Report
+{
+    protected function init()
+    {
+        parent::init();
+
+        $this->categoryId    = 'AIActivity_AIActivity';
+        $this->subcategoryId = 'General_Overview';
+        $this->name          = Piwik::translate($this->categoryId);
+        $this->order         = 80;
+
+        $this->metrics          = [];
+        $this->processedMetrics = [];
+
+        // only configure metrics if feature flag is enabled
+        if ($this->isEnabled()) {
+            $this->metrics = [
+                Metrics::METRIC_NB_VISITS_AI_AGENTS,
+                Metrics::METRIC_NB_UNIQ_VISITORS_AI_AGENTS,
+                Metrics::METRIC_NB_ACTIONS_AI_AGENTS,
+                Metrics::METRIC_MAX_ACTIONS_AI_AGENTS,
+                Metrics::METRIC_SUM_VISIT_LENGTH_AI_AGENTS,
+            ];
+        }
+    }
+
+    public function configureWidgets(WidgetsList $widgetsList, ReportWidgetFactory $factory)
+    {
+        $widgetsList->addWidgetConfig(
+            $factory->createWidget()
+                ->setName('AIActivity_Overview')
+                ->forceViewDataTable(Evolution::ID)
+                ->setAction('getEvolutionGraph')
+                ->setOrder(1)
+        );
+    }
```

Comment:
I would suggest to directly add the sparklines to the page. So it aligns with other overview pages.